tests/components/mqtt/test_climate.py

```diff
@@ -438,23 +438,23 @@ async def test_set_away_mode_pessimistic(hass, mqtt_mock):
     await hass.async_block_till_done()
 
     state = hass.states.get(ENTITY_CLIMATE)
-    assert state.attributes.get("preset_mode") is None
+    assert state.attributes.get("preset_mode") == "none"
```

Comment:
I'm not too familiar with how tests are typically created in HA but wouldn't it make more sense to assert it against "PRESET_NONE" instead of "none"? Because if "PRESET_NONE" were to change in the future, all these test-cases would fail and need to be changed manually.

Reply:
I'm not entirely sure; we don't expect those constants to change often if at all.

Reply:
Either is ok.